docs/glossary-121.md

```diff
@@ -0,0 +1,106 @@
+| **Active** | A program that is currently running.|
+| **Archive** | A program that has ended but the information is archived by an admin. (It cannot be run again but the user can read them and copy them into new program).|
+| **Users** | All individuals who have access to, and work on the 121 platform e.g. CVA officer, volunteers, Financial officers etc. A user can have different roles in different programs.|
+| **Roles** | Each user will be assigned a position for a specific program (e.g. CVA officer, volunteer, Financial officer etc.). This is done in the "Team" page of the program.|
+| **Permissions** | Each role has different authorization to preform different actions. e.g. only a Financial officer can approve a payment, CVA manager can validate and include People in the program. You can see which permissions are related to which role on the "Roles and permissions" page.|
+| **Overview**| The information and metrics about a program.|
+| **Team** | 121 users that will be working on this program and their assigned roles. |
+| **People Affected** | All registrations are listed in this page. The registration and inclusion or rejection of beneficiaries in the program will be displayed in this page under different status: registered, validated, included, declined, completed.|
+| **Payment** | The page were users can have an overview of the payment status and/or execute payment tranches towards included beneficiaries.|
+
+### Program Cards
+
+| Term                   | Definition                           |
+| :------------------    | :------------------------------------------------------------------- |
+| **Program Cards** | A small window providing an brief overview of the program details. |
+| **Targeted People Affected** | The amount of people/households that a program wishes to help.|
+| **Included People Affected** | The amount of people/households that have been receiving funds through a project.|
+| **Total Project budget** | The total amount of funds allocated for disbursement in a program.|
+| **Spent money** | The amount of cash disbursed or funds transferred to people/households in a program.|
+| **Program title** | The name of the program. |
+| **Last updated** | The most recent date and time when the system checked for changes made to a program.|
+| **Active** | Programs that are being created or already running.|
+| **Archived** | Rather than deleting, the user can archive a project, it is not active but its details are still available to view and can be copied.|
+
+### Registration Status
+
+| Term                   | Definition                           |
+| :------------------    | :------------------------------------------------------------------- |
+| **People Affected, PA** | Registered beneficiaries or households in the program, also referred to PA in 121. Depending on the SOPs, the status of the PA can vary: new, registered, validated, included, declined, completed, paused.|
```

Comment:
I believed we were moving away from People Affected. It is currently in the product though, not sure on how to proceed. @cassandradamico what do you think?

Reply:
People Affected, PA is also not a Registration status, so it is in the wrong section anyway

Reply:
@tijsziere I will keep PA for now, as this is not used outside of RCRC movement, for understanding matter. Will change whenever the terminilogy change in 121